lib/client-api/src/args.ts

```diff
@@ -1,9 +1,12 @@
 import { Args, ArgTypes } from '@storybook/addons';
+import { isPlainObject } from 'lodash';
 
 type ValueType = { name: string; value?: ObjectValueType | ValueType };
 type ObjectValueType = Record<string, ValueType>;
 
-const map = (arg: any, type: ValueType): any => {
+export const INCOMPATIBLE = Symbol('incompatible');
```

Comment:
This doesn't need to be exported as it's never returned from an exported function.